docs/welcome/office_hours.md

```diff
@@ -2,8 +2,14 @@
 
 kOps maintainers set aside one hour every other week for **public** office hours. This time is used to gather with community members interested in kOps. This session is open to both developers and users.
 
+### Regular office hours
+
 Office hours are hosted on a [zoom video chat](https://zoom.us/j/97072789944?pwd=VVlUR3dhN2h5TEFQZHZTVVd4SnJUdz09) on Fridays at [12 noon (Eastern Time)/9 am (Pacific Time)](http://www.worldtimebuddy.com/?pl=1&lid=100,5,8,12) during weeks with odd "numbers". To check this weeks' number, run: `date +%V`. If the response is odd, join us on Friday for office hours!
 
+### Informal office hours
+
+Since the regular office hours do not fit everyones schedule, kOps maintainers also host informal office hours. There is no fixed timeslot for these meetings, so watch #kops-dev on the kubernetes Slack, or the agenda mentioned below.
```

Comment:
I thought that for now we have a set time for the informal call.
```suggestion
Since the regular office hours do not fit everyones schedule, kOps maintainers also host informal office hours. Thursday at [9 am (Eastern Time)/6 am (Pacific Time) during weeks with even "numbers". To check this weeks' number, run: `date +%V`. If the response is even, join us on Thursday for office hours!
```

Reply:
ah okay. cool